applications/Preprocessor/lib/MeshSource.h

```diff
@@ -190,6 +191,36 @@ class MeshSource2 {
      */
     virtual const std::vector<Element>& getElements() = 0;
 
+    /**
+     * List of merges that should be applied to the mesh after initial
+     * construction.
+     *
+     * Each entry in the vector corresponds to a single merge operation. Where
+     * in each operation the coordinates identified by the indices on 'left' of
+     * the map are merged with the coordinates identified by the indices on the
+     * 'right' of the map. Each of these maps is fed to
+     * MergePlan#computeMergePlan for the actual merging.
+     *
+     * Note that unlike identifying coordinates beforehand (using Coord#nodeId),
+     * doing a merge via this route will prevent several problems in mesh
+     * construction. For example take two triangles in the configuration:
+     *
+     *  1 -- 0 -- 2
```

Comment:
where is the pbc?

Reply:
Added a documenting note this is for a rotation periodicity around node 0.